src/App.tsx

```diff
@@ -13,19 +22,20 @@ export const App: React.FC = () => {
             <label htmlFor="search-query" className="label">
               Search movie
             </label>
-
             <div className="control">
               <input
                 type="text"
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={event => setQuery(event.target.value.toLowerCase())}
```

Comment:
You shouldn't use toLowerCase() here. You don't give opportunity to enter big letters. You have already used it when chek coincidences in lines 12-13.